src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -1392,37 +1415,130 @@ export function intervalFiltersDisplayValue(values: DataFilterValue[]) {
             ? numericals[numericals.length - 1].end
             : undefined;
 
-    let displayValues: string[] = [];
+    let displayValues: any[] = [];
 
     if (numericals.length > 0) {
+        const startValue = formatValue(start!);
```

Comment:
it's odd that you use the ! even though above (1412) they are allowed to be undefined?

Reply:
the case when it is undefined is when the selected filters don't include numerical. e.g. in some bar charts you can have NA, a categorical filter in an otherwise numerical attribute. if you are only filtering for NA then you wouldn't have any numerical.

Reply:
what i meant is, i take the ! to mean you are telling typescript that it is NOT undefined.  but it can be undefined, which would suggest that formatValue can handle being passed undefined, in which case why the !      
sorry if i misunderstand

Reply:
ah I see. yes so formatValue can handle an undefined but if you look at the declaration it has multiple type signatures so it knows that a defined number input leads to a string. so you can look at `const startValue = formatValue(start!)` as equivalent to `const startValue = formatValue(start) as string;`